src/App.jsx

```diff
@@ -1,25 +1,28 @@
-// import "./index.css";
-import Background from "./component/Background/background";
-import Hero from "./component/Placeholder/PlaceHolder";
 import Selinelogo1 from "./assets/images/seline-logo1.png";
 import BackgroundImage from "./assets/images/background-image.jpg";
 import CountDown from "./Time/countdown";
 import Icon from "./component/social-media-icons/icon";
+import Hero from "./component/Placeholder/PlaceHolder";
```

Comment:
I was unable to run your code with the error below:
![error-savior](https://github.com/user-attachments/assets/70e0aa68-b0ef-4913-b5e5-46ee815bdad9)

Please resolve and revert